src/aequitas/flow/methods/preprocessing/label_flipping.py

```diff
@@ -282,7 +282,9 @@ def transform(
         Parameters
         ----------
         X : pd.DataFrame
-            Feature[s.loc[i]]ector.
```

Comment:
This change is already in master :thinking: Maybe this is going to need a rebase?